eclass/qt5-build.eclass

```diff
@@ -655,7 +655,9 @@ qt5_base_configure() {
 
 		# disable undocumented X11-related flags, override in qtgui
 		# (not shown in ./configure -help output)
-		-no-xkb -no-xrender
+		-no-xkb
+		$([[ ${QT5_MINOR_VERSION} -lt 8 ]] || $([[ ${QT5_MINOR_VERSION} == 8
```

Comment:
* why the nested `$(...)`? can't you do everything inside a single `[[ ]]`? (with parentheses if needed)
* use `-eq` instead of `==` for consistency
* change the commit msg, these options are used for *all* qtbase packages, not just {qtcore,qtgui}